Frameworks/CoreText/CTTypesetter.mm

```diff
@@ -127,13 +128,23 @@ static CFIndex _DoWrap(CTTypesetterRef ts, CFRange range, WidthFinderFunc widthF
 
         if (chars[curIndex] == 10 || (curIndex > 0 && chars[curIndex - 1] == 13 && curIndex - 1 >= lineStart)) {
             if ((curIndex > 0 && chars[curIndex - 1] == 13 && chars[curIndex] != 10)) {
-                curIndex--;
+                --curIndex;
+            }
+
+            if (curIndex > lineStart && chars[curIndex - 1] == ' ') {
```

Comment:
isspace would also catch CR, which we wouldn't want to accept here, and good catch, we're not currently handling tabs

---
In reply to: [74523721](https://github.com/Microsoft/WinObjC/pull/873#discussion_r74523721) [](ancestors = 74523721)